### PR TITLE
fix: changes to avoid sending logs to logdna

### PR DESCRIPTION
**Pull request by imprateeksh**
### Description

This is regarding the issue where we have to discontinue the usage of LogDNA and use IBM Cloud Logs. While the other solution is in progress but this change is to stop sending logs to logdna. 

### Release required?

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: module-assets/ci/run-tests.sh [hunk 1]
```diff
@@ -147,61 +147,10 @@ if [ ${IS_PR} == true ]; then
...
-      done
-
-      $test_cmd 2>&1 | tee "$log_location"
-
-    else
-      $test_cmd
-    fi
+    # Assign tags
+    export ICL_TAGS="${REPO_NAME}-PR${PR_NUM},commit-${COMMIT_ID},pipeline-${PIPELINE_ID},env-${ENV_TAG}"
+    echo "$ICL_TAGS" # Keeping this for now as this information will be required when using ICL
```
Comment (ocofaigh) [inline module-assets/ci/run-tests.sh:152]: Not sure we will use ICL here, but instead maybe leverage github checks. Remove this for now

**Review by ocofaigh — CHANGES_REQUESTED** [review]
see comment

**Review by ocofaigh — APPROVED** [review]

**Comment by imprateeksh** [comment]
Removed ICL reference and also removed some unused variables.

image

File: module-assets/ci/run-tests.sh (hunk 1 above)
Comment (imprateeksh) [reply module-assets/ci/run-tests.sh]: sure done

**Pull request merged**